position/sync/chooseOne.js

```diff
@@ -1,16 +1,15 @@
-const Postion = require('./position')
-const { chooseOnePositionType } = require('../../types')
+const Position = require('./position')
+const { CHOOSE_ONE } = require('../../types')
 
-function ChooseOne ({ poll, choice, reason, channel, recps, mentions }) {
```

Comment:
recps removed because Position() will handle this - read the poll, check the recps on that and copy them over